DCC.combined-schema.json

```diff
@@ -3,21 +3,42 @@
   "$id": "https://id.uvci.eu/DCC.combined-schema.json",
   "title": "EU DCC",
   "description": "EU Digital Covid Certificate",
-  "$comment": "Schema version 1.2.1",
-  "required": [
-    "ver",
-    "nam",
-    "dob"
-  ],
+  "$comment": "Schema version 1.3.0",
   "type": "object",
+  "oneOf": [
```

Comment:
@martin-lindstrom This is an official EU Regulation, so why not also include that in this schema?

Reply:
Preferred is indeed a constraint in the Issuance Business Rules (Postel's Law, 2nd part).
(Numerous) discussions in sub-group where I pointed out we should keep schema flexible (Postel's Law, 1st part) and apply constraints firstly in issuance, secondly in verification business rules.
End result was to keep the basic structure of array, but limit to 1 item, at least for current purposes.

Reply:
Yes just setting `maxItems = 1` is an option, also considered. `maxItems` operates at the level of the individual array. The oneOf/allOf construct operates at the level above that -> how the DCC payload in total is constructed.

> OK, a DCC with no v, r or t would then be ok according to the schema, but that is just a silly corner case.

Not even a corner case, won't happen at all. Why not? A certificate will only be issued for a medical event - v / t / r. If there is no v / t / r there is no certificate - at all. Thus if a certificate exists it should contain either a v / t / r. The only time it would not is if it is malformed (whether deliberately so or just through a badly broken issuer app). In these cases, the allOf / oneOf construct can rapidly filter out the malformed payload.

I saw there was mention later on of Postel's Law so I'll deal with the role of the DCC Schema and e.g. allOf/oneOf from both an Issuer and a Verifier perspective there.